frontend/src/v5/ui/controls/actionMenu/actionMenu.component.tsx

```diff
@@ -14,7 +14,7 @@
  *  You should have received a copy of the GNU Affero General Public License
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-import { ReactNode, useState } from 'react';
+import { ReactNode, useEffect, useState } from 'react';
```

Comment:
Are the changes in this file still necessary?

Reply:
Not anymore, I reverted them